libs/designsystem/src/lib/components/modal/services/modal.helper.spec.ts

```diff
@@ -233,6 +240,46 @@ describe('ModalHelper', () => {
           });
         });
 
+        describe(`drawer can interact with background`, () => {
+          beforeEach(async () => {
+            await openDrawer('Drawer On Presenting Element', undefined, 'medium', true);
+          });
+
+          afterEach(async () => {
+            await overlay.dismiss();
+          });
+
+          it(`body should have class '${ALLOW_BACKGROUND_SCROLL_CLASS_NAME}' if interactWithBackground is true`, () => {
+            expect(window.document.body.classList).toContain(ALLOW_BACKGROUND_SCROLL_CLASS_NAME);
+            expect(window.document.body).toHaveComputedStyle({
+              overflow: 'visible',
+            });
+          });
+
+          it(`Drawer close should remove '${ALLOW_BACKGROUND_SCROLL_CLASS_NAME}'`, async () => {
+            await overlay.dismiss();
+            expect(window.document.body.classList).not.toContain(
+              ALLOW_BACKGROUND_SCROLL_CLASS_NAME
+            );
+          });
+        });
+
+        describe(`Drawer can not interact with background`, () => {
+          beforeEach(async () => {
+            await openDrawer('Drawer On Presenting Element', undefined, 'medium', false);
+          });
+
+          afterEach(async () => {
+            await overlay.dismiss();
+          });
+
+          it(`body should not have class '${ALLOW_BACKGROUND_SCROLL_CLASS_NAME}' if interactWithBackground is false`, () => {
```

Comment:
Consider wording (surrounding `describe` block already contains the drawer state in regards to interactWithBackground):
```suggestion
          it(`body should not be scrollable`, () => {
```

Reply:
Fixed in next commit